src/libs/antares/study/simulation.h

```diff
@@ -64,6 +65,8 @@ class Simulation final
     */
     bool saveToFolder(const AnyString& folder) const;
 
+    void saveUsingWriter(Solver::IResultWriter::Ptr writer, const AnyString& folder) const;
```

Comment:
Current class and source file have a terrible name (**Simulation**).
Actually, it's all about **user's notes**, loading and saving them (modifying them is handled in : 
src/ui/simulator/windows/notes.cpp).
It can be misleading. So this class should renamed **UserNotes**. Same for source file.
The content of user's notes should be renamed from **comments** into **content**.

Reply:
Once "simulation" renamed, **saveUsingWriter** will be able to keep its name.